sdks/python/container/base_image_requirements.txt

```diff
@@ -67,7 +67,7 @@ pandas==0.23.4
 protorpc==0.11.1
 python-gflags==3.0.6
 setuptools<=39.1.0 # requirement for Tensorflow.
-tensorflow==1.11.0
+tensorflow==1.14.0
```

Comment:
Keep these two changes here, because previous versions are not compatible with py37 and py37 tests cannot pass.